black.py

```diff
@@ -126,6 +126,7 @@ class FileMode(Flag):
     AUTO_DETECT = 0
     PYTHON36 = 1
     PYI = 2
+    SINGLE_QUOTE = 4
```

Comment:
If you decide to keep the fork, note: https://github.com/ambv/black/commit/8ebbd268880f15834b70910a6dc61e1ee7596b7c

(you'll want to pick a higher number than 4; say 64 since we're unlikely to reach it any time soon)

Reply:
Oh nice, thanks @ambv -- we'll explore this new flag. Good compromise on `--skip-string-normalization`. 👍